poms/bom/pom.xml

```diff
@@ -332,6 +337,11 @@
                 <artifactId>camel-quarkus-log</artifactId>
                 <version>${camel-quarkus.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-microprofile-health</artifactId>
+                <version>${project.version}</version>
```

Comment:
Could you please replace `${project.version}` with `${camel-quarkus.version}`? This is to be able to use the BOM as a parent in user projects.

Reply:
I've fixed that up now.